code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -404,7 +404,17 @@ var/global/list/activated_medevac_stretchers = list()
 
 //bedroll
 /obj/structure/bed/bedroll
-	name = "bedroll"
-	desc = "bedroll"
+	name = "unfolded bedroll"
+	desc = "A rolled up bedroll, not too comfortable rolled up, you should probably unroll it before you sleep on it."
```

Comment:
Your description doesn't match the name, this is the unrolled bedroll, yet you say its rolled up.